packages/beta/src/types.ts

```diff
@@ -133,3 +134,360 @@ export interface RelationshipsFilter extends CreatedAndLastUpdatedTimeFilter {
    */
   labels?: LabelFilter;
 }
+
+export type ContextJobStatus = 'QUEUED' | 'RUNNING' | 'COMPLETED' | 'FAILED';
+
+export const ContextJobStatus = {
+  QUEUED: 'QUEUED' as ContextJobStatus,
+  RUNNING: 'RUNNING' as ContextJobStatus,
+  COMPLETED: 'COMPLETED' as ContextJobStatus,
+  FAILED: 'FAILED' as ContextJobStatus,
+};
+
+/**
+ * Contextualization job ID.
+ */
+export type ContextJobId = number;
+
+export interface EntityMatchingResponseBase {
+  /**
+   * User defined name of the model.
+   */
+  name?: string;
+  /**
+   * User defined description of the model
+   */
+  description?: string;
+  /**
+   * The status of the job.
+   */
+  status?: ContextJobStatus;
+  createdTime?: Date;
+  startTime?: Date;
+  statusTime?: Date;
```

Comment:
is seems that 
```
createdTime: Date;
  startTime: Date;
  statusTime: Date;
```
supposed to be required by the docs?

Reply:
I can see `jobId` and `status` is required in `#operation/entityMatchingPredictResults`.
is it required everywhere?. if not then you cannot use same `EntityMatchingResponseBase` everywhere.
if yes - then just a matter of fixing it here

Reply:
I'm using 
https://pr-1122.specs.preview.cogniteapp.com/v1.json.html#tag/Entity-matching 
as the source of truth.

Reply:
@chingyu-cognite not sure I am following.
those fields are required in a pr you referenced.
<img width="772" alt="Screenshot 2020-11-23 at 16 48 47" src="https://user-images.githubusercontent.com/10908415/99983202-d7e99880-2dab-11eb-89ca-f6a193ba1b46.png">